patches/woocommerce-force-sells.0001.fix.function-called-on-null.patch

```diff
@@ -0,0 +1,83 @@
+From b971735ed630c1e4d1d60db8b6142652d9404636 Mon Sep 17 00:00:00 2001
+From: Andres A <andres@netzstrategen.com>
+Date: Wed, 9 Feb 2022 10:49:24 +0100
+Subject: [PATCH] Fixed function call on null when cart does not exist in session.
+
+---
+ woocommerce-force-sells.php | 32 +++++++++++++++++++++++---------
+ 1 file changed, 23 insertions(+), 9 deletions(-)
+
+diff --git a/woocommerce-force-sells.php b/woocommerce-force-sells.php
+index 7c8856667..203a642f6 100644
+--- a/woocommerce-force-sells.php
++++ b/woocommerce-force-sells.php
+@@ -68,8 +68,8 @@ if ( ! class_exists( 'WC_Force_Sells' ) ) :
+ 			// Keep force sell data in the cart.
+ 			add_filter( 'woocommerce_get_cart_item_from_session', array( $this, 'get_cart_item_from_session' ), 10, 2 );
+ 			add_filter( 'woocommerce_get_item_data', array( $this, 'get_linked_to_product_data' ), 10, 2 );
+-			add_action( 'woocommerce_cart_loaded_from_session', array( $this, 'remove_orphan_force_sells' ) );
+-			add_action( 'woocommerce_cart_loaded_from_session', array( $this, 'maybe_remove_duplicate_force_sells' ) );
++			add_action( 'woocommerce_cart_loaded_from_session', array( $this, 'remove_orphan_force_sells' ), 10, 1 );
++			add_action( 'woocommerce_cart_loaded_from_session', array( $this, 'maybe_remove_duplicate_force_sells' ), 10, 1 );
```

Comment:
You can recreate the patch file after merging the file project PR into master.  Delete this file first, so that the new file gets the same number 0001.